addons/snailmail/models/snailmail_letter.py

```diff
@@ -455,9 +455,19 @@ def _is_valid_address(self, record):
         required_keys = ['street', 'city', 'zip', 'country_id']
         return all(record[key] for key in required_keys)
 
+    def _get_cover_address_split(self):
+        if self.country_id.code == 'DE':
+            # https://help.pingen.com/en/faq-post/adressanforderungen-deutsche-post#unerlaubter-inhalt-plz-zeile
+            address_split = self.partner_id.with_context(show_address=True, lang='de_DE').display_name.split('\n')
```

Comment:
If the `de_DE` language isn't activated on the database, this will crash.
I don't think we can safely assume that it is installed :thinking:.

Reply:
My bad, good catch!